### PR TITLE
Plane: Added ground effect compensation for quadplanes

### PR DESCRIPTION
**Pull request by tridge**
This can help on some quadplanes where the fuselage design can lead to large levels of baro ground effect

### CODE REVIEW

File: ArduPlane/quadplane.cpp [hunk 1]
```diff
@@ -328,8 +328,8 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
...
+    // @Bitmask: 0:LEVEL_TRANSITION,1:ALLOW_FW_TAKEOFF,2:ALLOW_FW_LAND,3:VTOL_TAKEOFF_REF,4:USE_FW_APPROACH,5:USE_QRTL,6:USE_GOVERNOR,7:FORCE_QASSIST,8:MTRS_ONLY_QASSIST,9:AIRMODE_ON_ARM,10:DISARMED_YAW_TILT,11:DELAY_SPOOLUP,12:GROUND_EFFECT_COMP
```
Comment (WickedShell) [inline ArduPlane/quadplane.cpp:332]: I'm sorry, but the caps locking of these are far worse, and less readable. `Level transition` is no longer then `LEVEL_TRANSITION` and in fact on most displays will be shorter if you aren't using a mono space font.

Comment (peterbarker) [reply ArduPlane/quadplane.cpp:332]: I'm a bit surprised to see this.  My take-away from the DevCall conversation was that we'd be using e.g. "Level Transition", "Allow FW Takeoff" etc.... with or without caps, followed up by some way of providing an extended description for each bit.

**Comment by tridge** [comment]
make on by default

**Review by peterbarker — APPROVED** [review]

**Comment by CallanDaniel** [comment]
Any chance we can get this into the next Plane stable release? @tridge

**Pull request merged**